doc/supabase-configuration.md

```diff
@@ -0,0 +1,49 @@
+# Supabase Configuration
+
+## Remote Instance Setup
+
+You can create a remote supabase using the following script:
+```sh
+make supabase-remote-init
+```
+
+The script will prompt you for the project configuration and will apply migrations and deploy edge functions.
+
+
+## Manual Remove Instance Link
+
+If you already created the supabase instance, you can link the instance manually using the following commands:
+First, log into your supabase account:
+
+```sh
+npx supabase login
+```
+
+Now, link this project to the local supabase instance. You'll be asked to enter the database password.
+```sh
+npx supabase link --project-ref ********************
+```
+
+Then, apply the migrations on it:
+```sh
+npx supabase db push
+npx supabase functions deploy
+```
+
+Finally, create the `.env.production.local` file with your supabase configuration:
+
+```sh
+VITE_SUPABASE_URL=<instance_url>
+VITE_SUPABASE_ANON_KEY=<instance_anon_token>
+```
```

Comment:
This is not clear whether this is applicable when you deploy from your local machine or from a CI/CD. I believe this is applicable to the 1st case and not the 2nd (but correct me if I'm wrong!).
If that's the case, we should explain this, and also explain how to do it from a CI/CD.